data/architecture.yml

```diff
@@ -10,7 +10,7 @@ questions:
     -
         question: 'What is the Symfony mail service transport parameter name?'
         answers:
-            - {value: mailer.transport,   correct: true}
+            - {value: swiftmailer.transport,   correct: true}
```

Comment:
@mickaelandrieu 

parameter name is:
```yml
transport: '%mailer_transport%'
```
http://symfony.com/doc/current/cookbook/email/email.html


But the service name is "swiftmailer"
https://github.com/symfony/swiftmailer-bundle/blob/6b7316b3471fe28c35c047b4bf17a2ff946f5f66/Resources/config/swiftmailer.xml#L33-L35